setup.cfg

```diff
@@ -0,0 +1,19 @@
+[metadata]
+name = skt
+description = "Sonic Kernel Testing"
+long_description = file: README.md
+version = 0.0.1
```

Comment:
Would you be open to just having `version = 1` here? I'm aware of semantic versioning, but, I find that so many numbers in version are not needed until at least half a dozen releases. Meanwhile we'll be having major changes in the code and version number changes won't reflect that. We can always add dots and number when we need them.

Reply:
Sure. ;)